examples/howto/plot_hnn_animation.py

```diff
@@ -0,0 +1,37 @@
+"""
+================================
+XX. Modifying local connectivity
+================================
+
+This example demonstrates how to animate HNN simulations
+"""
+
+# Author: Nick Tolley <nicholas_tolley@brown.edu>
+
+
+###############################################################################
+from hnn_core import jones_2009_model, simulate_dipole
+from hnn_core.network_models import add_erp_drives_to_jones_model
+from hnn_core.viz import NetworkPlot
```

Comment:
```suggestion
from hnn_core.viz import NetworkPlotter
```

Reply:
I'm guessing the name of this class got changed somewhere along the way and you forgot to update it here?